youtube_dl/extractor/qqmusic.py

```diff
@@ -112,15 +117,35 @@ def _real_extract(self, url):
         self._check_formats(formats, mid)
         self._sort_formats(formats)
 
-        return {
-            'id': mid,
-            'formats': formats,
-            'title': song_name,
-            'upload_date': publish_time,
-            'creator': singer,
-            'description': lrc_content,
-            'thumbnail': thumbnail_url,
-        }
+        actual_lrc_lyrics = self._filter_lrc(lrc_content)
+
+        if actual_lrc_lyrics:
+            return {
+                    'id': mid,
+                    'formats': formats,
+                    'title': song_name,
+                    'upload_date': publish_time,
+                    'creator': singer,
+                    'description': lrc_content,
+                    'subtitles': {
+                        'zh-CN': [{
+                            'ext': 'lrc',
+                            'data': actual_lrc_lyrics,
+                            }]
+                        },
+                    'thumbnail': thumbnail_url,
+                    }
+        else:
+            return {
+                    'id': mid,
+                    'formats': formats,
+                    'title': song_name,
+                    'upload_date': publish_time,
+                    'creator': singer,
+                    'description': lrc_content,
+                    'thumbnail': thumbnail_url,
+                    }
```

Comment:
Just update the dictionary with `subtitles` key when `.lrc` detected. There is no need in code duplication. Also replace `zh-CN` with something more generic like `origin`.